index.html

```diff
@@ -253,11 +253,6 @@ <h1>Skillcrush 105 - Advice for Newbies!</h1>
                 <a href="advice/simon-advice.html" target="_blank">Simon's Advice</a><br>
                 <a href="advice/beth-advice.html" target="_blank">Beth's Advice</a><br>
                 <a href="advice/kimberly-advice.html" target="_blank">Kimberly's Advice</a><br>
-                <a href="advice/mariko-advice.html" target ="_blank"> Mariko's Advice</a><br>   
-                <a href="advice/elisefolk-advice.html" target ="_blank"> Elise's Advice</a><br>  
-                <a href="advice/jennylynn-advice.html" target ="_blank"> Jenny Lynn's Advice</a><br>         
-                <a href="advice/karynpaige-advice.html" target="_blank"> Karyn's Advice</a><br>  
-                <a href="advice/steve-m-advice.html" target="_blank">Steve Manning's Advice</a><br>
```

Comment:
If you look at lines 256-260, it looks like you removed other students advice files. We want to preserve everyone's work so these will need to be added back in. We would also like for you to add a link that links to your personal advice page.